src/helpers/toast.ts

```diff
@@ -0,0 +1,34 @@
+import { toast as soner } from "vue-sonner"
+
+export type ToastOptions = {
+  type?: "success" | "error" | "warning" | "info"
+  duration?: number
+}
+
+export const toast = (message: string, option?: ToastOptions) => {
+  soner(message, {
+    ...option,
+  })
+}
+
+/*
+ * Legacy support for toast.success, toast.error, and toast.warning
+ */
+
+toast.success = (message: string, option?: ToastOptions) => {
+  soner.success(message, {
+    ...option,
```

Comment:
Do we need to give options here since type is already hardcoded here? Do we have duration support in our old toast implementation? If not, we can remove this.